examples/ui-demo/src/app/config.tsx

```diff
@@ -0,0 +1,82 @@
+import { AuthCardHeader } from "@/components/shared/AuthCardHeader";
+import { alchemy, arbitrumSepolia } from "@account-kit/infra";
+import { cookieStorage, createConfig } from "@account-kit/react";
+import { AccountKitTheme } from "@account-kit/react/tailwind";
+import { QueryClient } from "@tanstack/react-query";
+
+export type Config = {
+  auth: {
+    showEmail: boolean;
+    showExternalWallets: boolean;
+    showPasskey: boolean;
+    addPasskey: boolean;
+  };
+  ui: {
+    theme: "light" | "dark";
+    primaryColor: {
+      dark: string;
+      light: string;
+    };
+    borderRadius: AccountKitTheme["borderRadius"];
+    illustrationStyle: "outline" | "linear" | "filled" | "flat";
+    logoLight:
+      | {
+          fileName: string;
+          fileSrc: string;
+        }
+      | undefined;
+    logoDark:
+      | {
+          fileName: string;
+          fileSrc: string;
+        }
+      | undefined;
+  };
+  supportUrl?: string;
+};
+
+export const DEFAULT_CONFIG: Config = {
+  auth: {
+    showEmail: true,
+    showExternalWallets: false,
+    showPasskey: true,
+    addPasskey: true,
```

Comment:
this is false right?

Reply:
or will be...

Reply:
I guess fine to do it in another PR!

Reply:
ah yea I think that's what Iyk and Rob are gonna tackle changing